README.md

```diff
@@ -1,15 +1,73 @@
-# GitHub Repo Template
+# Audio ADC - LabVIEW
+
+## Overview
+
+'Audio ADC - LabVIEW' is a MeasurementLink based plugin that enables users to perform measurements of Audio Analog to Digital Converters.
```

Comment:
Rephrase as "Audio ADC - LabVIEW" is a MeasurementLink LabVIEW plugin for making measurements for Audio ADC's